junit-platform-engine/src/main/java/org/junit/platform/engine/discovery/DiscoverySelectors.java

```diff
@@ -241,6 +242,21 @@ public static ClasspathResourceSelector selectClasspathResource(String classpath
 	}
 
 	/**
+	 * Create a {@code ModuleSelector} for the supplied module name.
+	 *
+	 * <p>The unnamed module is not supported.
+	 *
+	 * @param moduleName the module name to select; never {@code null} and
+	 * never blank
```

Comment:
I think we usually use `never {@code null} or blank`.

Reply:
Copy and paste error by me.